README.md

```diff
@@ -136,8 +136,28 @@ To clone the Choreo documentation GitHub repository and run the site on your loc
       ```shell
       $ pip3 install -r requirements.txt
       ```
+### Step 4 - Build Project
+1. To build the code navigate to the en folder and run the following command:
 
-### Step 4 - Run MkDocs
+    ```shell
+    $ ./serve.sh
+    ```
+This script will check spellings and incorrect links. 
+
+Incorrect Spellings will be highlighted in red. You can take one of the resolution paths:
+- You can choose the correct suggested spelling. 
+- "Ignore" will ignore that word and not ask about it again in the current run. If you re-run the command again though, it will appear.
```

Comment:
```suggestion
- Proceed with "Ignore" to skip the word and not be prompted about it again during the current run. However, it will reappear if you rerun the command.
```